doc/spec/estimation/forest.rst

```diff
@@ -58,10 +58,10 @@ supports fit and predict. The fit function needs to also support sample weights,
 If it does not, then we provided a weighted model wrapper :py:class:`~econml.ortho_forest.WeightedModelWrapper` that
 can wrap any class that supports fit and predict and enables sample weight functionality. This is done either
 by re-sampling the data based on the weights and then calling fit and predict, or, in the case of square losses of
-linear function classes, by re-scaling the features and labels appropriately based on the weights.
+linear function classes, by re-scaling the features and labels appropriately based on the weights::
 
-    >>> est = ContinuousTreatmentOrthoForest(model_y=WeightedModelWrapper(Lasso(), sample_type=sample_type),
-    ...                                      model_t=WeightedModelWrapper(Lasso(), sample_type=sample_type))
+    est = ContinuousTreatmentOrthoForest(model_y=WeightedModelWrapper(Lasso(), sample_type=sample_type),
```

Comment:
If we added a testcode below, we should also add one here. Also we should replace the models here with `econml.sklearn_extensions.linear_model.WeightedLasso`

Reply:
Okay.  I think the reason I avoided it was that it seemed like perhaps overkill to include all of the necessary imports in this introductory section, although we can include them in a hidden testcode block if we want to test this without adding too much content.

Reply:
Yeap, I would include in a hidden code block, just to make sure we have not mistake in an opening paragraph

Reply:
Done.  Hilariously, there was a mistake here which was caught by running it: for OrthoForests the parameters are called `model_T` and `model_Y` rather than `model_t` and `model_y`.

Reply:
It's obvious we should be testing every single line :) Next PR we'll add tests for the tests :)